spyderlib/plugins/__init__.py

```diff
@@ -124,6 +126,14 @@ def __init__(self, *args, **kwargs):
         if sys.platform == 'darwin':
             self.setStyleSheet(self.DARWIN_STYLE)
 
+        # Needed for the installation of the event filter
+        self.title = args[0]
+        self.main = args[1]

```

Comment:
when create_dockwidget is called the args passed are the title and the parent (the main window in this case)

I agree is not so clear, but I am not so sure how to make it better...